docs/components/modeler/desktop-modeler/index.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This feels a little bare. Perhaps we could have brief outlines of BPMN/DMN/Forms with links to these, or some next steps?

Reply:
Thanks for your comment. @YanaSegal Shared the same feedback on a different channel.

I suggested to add a short features list, as well as "getting started" to the page:

![image](https://github.com/camunda/camunda-platform-docs/assets/58601/beaaf42c-6f82-4f35-b33e-288c013b2a0a)

What do you think?

Reply:
Added a features list as shown in above screenshot (https://github.com/camunda/camunda-platform-docs/pull/2029/commits/18bb8e41295045185533573701d936309001c7f7). Let's incorporate this and iterate on the page as we see fit.